src/main/controlMod.F90

```diff
@@ -1192,7 +1199,10 @@ subroutine control_print ()
        write(iulog, *) '    use_fates_nocomp = ', use_fates_nocomp
        write(iulog, *) '    use_fates_sp = ', use_fates_sp
        write(iulog, *) '    use_fates_luh= ', use_fates_luh
+       write(iulog, *) '    use_fates_lupft= ', use_fates_lupft
```

Comment:
Probably something for later, but these could use better descriptions to describe what they are all about. With more options it's becomes more difficult to know what they all do.

Reply:
New issue: #2649